pkg/deploy/strategy/support/lifecycle.go

```diff
@@ -373,6 +373,7 @@ func makeHookPod(hook *deployapi.LifecycleHook, rc *kapi.ReplicationController,
 				Name:      mount.Name,
 				ReadOnly:  mount.ReadOnly,
 				MountPath: mount.MountPath,
+				SubPath:   mount.SubPath,
```

Comment:
Can't we just copy the whole object? there should be a deep copy method generated somewhere, right?